docs/elements/hx-accordion-panel/index.html

```diff
@@ -8,7 +8,7 @@
 {% extends 'element.njk' %}
 {% block content %}
   <section>
-    <dl class="hxBox-md metadata hxList">
+    <dl class="hxBox hxMd metadata hxList">
```

Comment:
Can we make this part of [`.metadata`](https://github.com/rackerlabs/helix-ui/blob/master/docs/docs.less#L92), instead of having to replace all these classes again in the future?

```less
@import 'components/box';

/* ... */

.metadata {
  #Box.md();

  /* ... */
}
```